### PR TITLE
PlaceCard placeholder

### PR DESCRIPTION
**Pull request by lukewalczak**
ezgif com-video-to-gif 1

Closes: #118

### CODE REVIEW

File: src/Placeholder/connect.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import * as React from 'react';
+
+type Props = {
+  onReady?: boolean,
+  children: React.Node,
+  animate?: string,
+  customAnimate?: () => void,
+};
+
+const connect = (PlaceholderComponent: any) => {
```
Comment (RobinCsl) [inline src/Placeholder/connect.js]: Could you change the type `any` to something more descriptive? I guess `React.Node` should work.

File: android/app/src/main/java/com/storybooknative/MainApplication.java [hunk 2]
```diff
@@ -22,7 +23,8 @@ public boolean getUseDeveloperSupport() {
     @Override
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
-          new MainReactPackage()
+          new MainReactPackage(),
+            new RNShimmerPackage()
```
Comment (RobinCsl) [inline android/app/src/main/java/com/storybooknative/MainApplication.java]:
```suggestion
          new RNShimmerPackage()
```

File: package.json [hunk 3]
```diff
@@ -33,9 +33,10 @@
   "dependencies": {
     "@kiwicom/orbit-components": "^0.23.0",
     "@kiwicom/orbit-design-tokens": "^0.2.5",
+    "intl": "^1.2.5",
+    "react-native-shimmer": "^0.4.1",
```
Comment (RobinCsl) [inline package.json]: I don't know how I feel about adding new libraries, especially native ones.

Did you try with `Animated`?

Comment (lukewalczak) [reply package.json]: To get a `shimmer` effect we need `gradient`.  
* Creating our own gradient probably will produce a lot of `View`s which with animations can have performance implications. 
* [Ready gradient lid](https://github.com/react-native-community/react-native-linear-gradient) also has to be linked

File: src/PlaceCard/PlaceCard.stories.js [hunk 4]
```diff
@@ -5,11 +5,47 @@ import { storiesOf } from '@storybook/react-native';
 import { boolean, text, withKnobs } from '@storybook/addon-knobs';
 
 import PlaceCard from './PlaceCard';
+import Placeholder from './Placeholder';
 
 const noop = () => {};
```
Comment (FilipMessa) [inline src/PlaceCard/PlaceCard.stories.js]: What about `@storybook/addon-actions`?

**Pull request merged**